streams/src/test/java/org/apache/kafka/streams/processor/internals/TaskManagerTest.java

```diff
@@ -2097,105 +1909,9 @@ public void shouldPinOffsetSumToLongMaxValueInCaseOfOverflow() throws Exception
         assertThat(taskManager.taskOffsetSums(), is(expectedOffsetSums));
     }
 
-    @Test
-    public void shouldCloseActiveUnassignedSuspendedTasksWhenClosingRevokedTasks() {
```

Comment:
I believe you cannot just delete all the tests that contain `tryToCompleteRetoration()`. You need to rewrite them. Let me know if you need help.